gulpfile.babel.js

```diff
@@ -139,4 +139,8 @@ gulp.task('build_client', () => {
   }, (err, stats) => {
       // FIXME(cs-lee) save log in file
   });
+
+  process.on('SIGINT', () => {
```

Comment:
I'm not sure if this is a right way.
Can you ensure that the other daemon or service is stopped safely?